randovania/games/dread/json_data/Cataris.txt

```diff
@@ -791,6 +799,8 @@ Extra - asset_id: collision_camera_014
   * Extra - right_shield_def: None
   > Door to Tall Magnet Walls Access
       Trivial
+  > Tile Group (POWERBEAM)
+      Speed Booster
```

Comment:
I know trickless.. But IBJ would work or not?

Reply:
Yeah, it would. forgot

Reply:
But that can be added once we go through adding the tricks

Reply:
Added this now